libs/model/src/services/session/transferOwnership.ts

```diff
@@ -34,30 +34,3 @@ export async function transferOwnership(
     if (updated > 0) return found?.User;
   }
 }
-
-// TODO: subscribe to AddressOwnershipTransferred event
```

Comment:
Why do we need to subscribe to the event?

Reply:
this is the old comment that triggered this ticket 😆

Reply:
Ah my bad lol I read this as an addition not deletion :)